src/NativeScript/ObjC/ObjCMethodCall.h

```diff
@@ -46,7 +46,8 @@ class ObjCMethodCall : public FFICall {
 
 private:
     ObjCMethodCall(JSC::VM& vm, JSC::Structure* structure)
-        : Base(vm, structure) {
+        : Base(vm, structure)
+        , _hasErrorOutParameter{ false } {
```

Comment:
I don't really mind initializing it in the constructor, I just feel queasy from the curly bracket syntax. I'm used to the round brackets syntax we employ elsewhere in the codebase :grinning: